script/rummager_export.rb

```diff
@@ -18,26 +22,39 @@
   Whitehall.searchable_classes_for_government_index
 end
 
-logger.info "Counting docs to index..."
-counts_by_class = classes_to_index.each_with_object({}) do |klass, hash|
-  count = klass.searchable_instances.count
-  logger.info("%20s: %d" % [klass.name, count])
-  hash[klass] = count
+id_groups = []
+classes_to_index.each do |klass|
+  id_groups += klass.searchable_instances.pluck(:id).each_slice(1_000).map do |id_group|
+    [klass, id_group]
+  end
 end
 
-total_count = counts_by_class.values.inject(&:+)
+def export_classes(classes_to_index, id_groups, &block)
+  if export_directory = ENV["EXPORT_DIRECTORY"]
+    export_directory = Pathname.new(export_directory).expand_path
+    export_directory.mkpath
```

Comment:
It won't.  Perhaps it should just error if the directory exists and is not empty?